src/kanata/mod.rs

```diff
@@ -340,7 +340,7 @@ impl Kanata {
         Ok(Arc::new(Mutex::new(Self::new(args)?)))
     }
 
-    fn do_live_reload(&mut self) -> Result<()> {
+    fn do_live_reload(&mut self, tx: &Option<Sender<ServerMessage>>) -> Result<()> {
```

Comment:
This isn't being checked in CI yet (good reminder for me to add it), but I believe this would give a warning when compiling without TCP support - i.e. with the flag `--no-default-features`. This can be done by changing the parameter name to `_tx`.